front-end/src/components/newsfeed-view/Header.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { getDate, formatDate } from '../../utils/utils';
+import DateHeader from './DateHeader';
+import './Header.css';
+
+
+const Header = props => (
+    <div className="header">
+        <h2>TODAY IN VANTAA</h2>
```

Comment:
Probably have a service to inject texts instead of hard coding things like this, ofc for now this is okay. but do have a strategy in mind.